.evergreen/config.yml

```diff
@@ -968,24 +807,12 @@ buildvariants:
      - name: "test-latest-sharded_cluster"
      - name: "test-latest-standalone"
      - name: "test-latest-load_balancer"
+     - name: "test-6.0-replica_set"
+     - name: "test-6.0-sharded_cluster"
+     - name: "test-6.0-standalone"
```

Comment:
What about 4.0 - 5.0? Should we add those too?

Reply:
Done